broker/src/main/java/io/camunda/zeebe/broker/partitioning/startup/steps/PartitionRegistrationStep.java

```diff
@@ -40,7 +40,7 @@ public ActorFuture<PartitionStartupContext> shutdown(final PartitionStartupConte
     final var partitionId = context.partitionMetadata().id().id();
     context.diskSpaceUsageMonitor().removeDiskUsageListener(context.zeebePartition());
     context.brokerHealthCheckService().removeMonitoredPartition(partitionId);
-    // TODO: Find a way to remove the partition from the topology manager
+    context.topologyManager().removePartition(partitionId);
```

Comment:
:question: This removes a failed bootstrap partition from topology manager. So it is not observable via `zbctl status` that this partition is DEAD. Right?

Reply:
Probably executing `topologyManager.onHealthChanged(partitionId, HealthStatus.DEAD);` in PartitionManagerImpl only after the partition is stopped would solve this.

Reply:
Yeah good catch, I'll change this as suggested :+1: